src/_layouts/default.html

```diff
@@ -10,20 +10,20 @@
     {% include header.html %}
 
     <!-- Page Content -->
-    <div class="container-fluid contents">
+    <div class="contents">
       <!-- Content Row -->
-      <div class="row">
+      <div class="lavish-row">
 
         <!-- Sidebar Column -->
-        <div id="side-nav-container" class="col-sm-3">
+        <div id="side-nav-container" class="col-md-3">
           {% include sidebar.html %}
         </div>
 
         {% if page.url == '/' %}
         {% comment %} Homepage {% endcomment %}
 
         <!-- Homepage Content Column -->
-        <div class="col-sm-12 main-contents">
+        <div class="col-md-12 main-contents">
```

Comment:
I don't understand why the change from `sm` to `md` here and elsewhere. Should we always use `sm` by default and then only override for bigger screen? (I thought that's the philosophy of bootstrap's responsive design.)

Reply:
Because a [new tier was added to BS 4](https://getbootstrap.com/docs/4.0/migration/#grid-system), so every tier has been bumped up one level:`sm` -> `md` -> `lg` -> `xl`.

Reply:
(Thanks for the feedback. I'm heading out, so I'll respond to you other comments later.)

Reply:
Just to be clear: the media breakpoints of both the current and this new version of the site are the same (e.g., 710px is one breakpoint for the home page). It is just that break point names have changed.

Reply:
Thanks for clarifying!